.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -286,7 +286,7 @@ jobs:
               --flush-ct
 
       - name: Rotate IPsec Key & Test (${{ join(matrix.*, ', ') }})
-        uses: cilium/cilium/.github/actions/conn-disrupt-test@d42be92482e5568f5899626b95df369b39d03276
```

Comment:
@jschwinger233 do we want to update this GH action on older branches as well? This means that any changes done in `.github/actions/conn-disrupt-test` in main will affect older branches.

Reply:
I believe we had this discussion before and agreed on not referencing action by sha, but that decision was made after backport 1.14 got merged. Therefore I'd suggest doing the same as 1.13 backport here. (Sorry but I can't reach my laptop until Sunday night)

Reply:
Since we have backported action.yaml to each 1.1x branch, maybe we can reference it by branch name to prevent changes on main branch from affecting others?

Reply:
Cool, please see https://github.com/cilium/cilium/pull/29252